Content.Server/Voting/VoteCommands.cs

```diff
@@ -129,6 +179,85 @@ public CompletionResult GetCompletion(IConsoleShell shell, string[] args)
             var n = args.Length - 1;
             return CompletionResult.FromHint(Loc.GetString("cmd-customvote-arg-option-n", ("n", n)));
         }
+
+        // Sends the payload's message. 
+        private async void WebhookMessage(WebhookPayload payload, string id)
+        {
+            if (id == string.Empty)
+            {
+                var request = await _httpClient.PostAsync($"{_webhookUrl}?wait=true",
+                    new StringContent(JsonSerializer.Serialize(payload), Encoding.UTF8, "application/json"));
+
+                var content = await request.Content.ReadAsStringAsync();
+                webhookId = (string) JsonNode.Parse(content)?["id"]!;
+            } 
+
+            await _httpClient.PatchAsync($"{_webhookUrl}/messages/{id}",
+                new StringContent(JsonSerializer.Serialize(payload), Encoding.UTF8, "application/json"));
+        }
+
+        public struct WebhookPayload
+        {
+            [JsonPropertyName("username")]
+            public string Username { get; set; } = "";
+
+            [JsonPropertyName("avatar_url")]
+            public string AvatarUrl { get; set; } = "";
+
+            [JsonPropertyName("embeds")]
+            public List<Embed>? Embeds { get; set; } = null;
+
+            public WebhookPayload() { }
+        }
+
+        public struct Embed
```

Comment:
Having a `Timestamp` would be good

Reply:
Discord messages are timestamped when they're sent, I don't see why it needs one